app/redux/selectors/SetsSelectors.js

```diff
@@ -388,7 +388,9 @@ const getBestEverOfMetric = (state, set, metricFunction, isMax=true) => {
     const matchedSets = historySets.concat(workoutSets).filter(historySet => areSetsComparable(historySet, set));
     
     let metrics = matchedSets.map((matchedSet) => {
-        return metricFunction(matchedSet);
+        if (!SetUtils.isDeleted(matchedSet)) {
+            return metricFunction(matchedSet);
```

Comment:
- [ ] Isn't map always supposed to return a value? What happens here when you don't?